modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/StorageHashIndexDescriptor.java

```diff
@@ -33,9 +33,7 @@
  * @see HashIndexStorage
  */
 public class StorageHashIndexDescriptor implements StorageIndexDescriptor {
-    /**
-     * Descriptor of a Hash Index column.
-     */
+    /** Descriptor of a Hash Index column. */
```

Comment:
Please revert all these useless javadoc updates

Reply:
I didn't see anywhere that this was prohibited.

Reply:
Excuse me? Please don't introduce useless changes that only please you, you make it harder to review your PRs.
I see that you are very motivated to update javadocs to a uniform style. That's a good thing. But please do it once in a separate PR instead of polluting every other PR. Thank you!

Reply:
fix it